src/components/toast-notification-manager/stories/common.ts

```diff
@@ -0,0 +1,121 @@
+import "#package/components/button/daikin-button";
+import "#package/components/toast-notification-manager/daikin-toast-notification-manager";
+import type {
+  DaikinToastNotificationManager,
+  ToastPosition,
+} from "#package/components/toast-notification-manager/daikin-toast-notification-manager";
+import "#package/components/toast-notification/daikin-toast-notification";
+import type { DaikinToastNotification } from "#package/components/toast-notification/daikin-toast-notification";
+import type { ElementProps } from "#storybook";
+import type { Meta, StoryObj } from "@storybook/web-components";
+import { LitElement, html } from "lit";
+import { customElement, property, state } from "lit/decorators.js";
+import { ifDefined } from "lit/directives/if-defined.js";
+import { repeat } from "lit/directives/repeat.js";
+
+export type ToastCloseEvent = CustomEvent<{ target: DaikinToastNotification }>;
+
+@customElement("sb-toast-notification-container")
+export class SBToastNotificationContainer extends LitElement {
+  @property({ type: String, reflect: true })
+  position: ToastPosition = "bottom-right";
+
+  @property({ type: Number, reflect: true, attribute: "item-duration" })
+  itemDuration: number | null = null;
+
+  @property({ type: Boolean, attribute: "is-vrt" })
+  isVrt: boolean = false;
+
+  @state()
+  private _items: string[] = [];
+
+  @state()
+  private _index = 0;
+
+  private _handleClick() {
+    const newIndex = this._index + 1;
+
+    this._items = [...this._items, `toast ${newIndex}`];
```

Comment:
I think the latest item should be at the front.

```suggestion
    this._items = [`toast ${newIndex}`, ...this._items];
```